editor.planx.uk/src/theme.ts

```diff
@@ -57,13 +57,13 @@ const DEFAULT_PALETTE: Partial<PaletteOptions> = {
   action: {
     selected: "#F8F8F8",
     focus: "#FFDD00",
-    disabled: "#B4B4B4",
+    disabled: "#949494",
```

Comment:
This meets the 3:1 requirement - https://webaim.org/resources/contrastchecker/?fcolor=949494&bcolor=FFFFFF